cli/cli.go

```diff
@@ -1,35 +1,40 @@
 package cli
 
 import (
-	"github.com/env0/terratag/errors"
+	"fmt"
 	"log"
 	"os"
 	"strconv"
 	"strings"
+
+	"github.com/env0/terratag/errors"
 )
 
-func InitArgs() (string, string, bool, bool) {
-	var tags string
-	var dir string
-	var skipTerratagFiles string
-	var isSkipTerratagFiles bool
+type Args struct {
+	Tags                string
+	Dir                 string
+	SkipTerratagFiles   string
+	IsSkipTerratagFiles bool
+	Verbose             bool
+	Rename              bool
+}
 
+func InitArgs() (Args, bool) {
+	args := Args{}
 	isMissingArg := false
 
-	tags = setFlag("tags", "")
-	dir = setFlag("dir", ".")
-	skipTerratagFiles = setFlag("skipTerratagFiles", "true")
+	args.Tags = setFlag("tags", "")
+	args.Dir = setFlag("dir", ".")
+	args.IsSkipTerratagFiles = booleanFlag("skipTerratagFiles", true)
+	args.Verbose = booleanFlag("verbose", false)
+	args.Rename = booleanFlag("rename", true)
```

Comment:
I'd change this flag to be `overwrite` and default to `false`

Reply:
imho `overwrite` boolean is a confusing term. `overwrite` true makes sense. but in false, i would expect the file not to change, but currently it disappears, and two other files appear in it's place... so i'm not in love with `rename` but i would avoid `overwrite`, at least in my association of the term, it's confusing.

Reply:
That's fair.